data/virt_autotest/host_autoyast/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm t="boolean">false</confirm>
+      <final_reboot t="boolean">true</final_reboot>
+      <second_stage t="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum t="boolean">true</accept_file_without_checksum>
+      <accept_non_trusted_gpg_key t="boolean">true</accept_non_trusted_gpg_key>
+      <accept_unknown_gpg_key t="boolean">true</accept_unknown_gpg_key>
+      <accept_unsigned_file t="boolean">true</accept_unsigned_file>
+      <accept_verification_failed t="boolean">true</accept_verification_failed>
+      <import_gpg_key t="boolean">true</import_gpg_key>
+    </signature-handling>
+  </general>
+  <report>
+    <errors>
+      <show t="boolean">true</show>
+      <timeout t="integer">20</timeout>
+      <log t="boolean">true</log>
+    </errors>
+    <messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">5</timeout>
+      <log t="boolean">true</log>
+    </messages>
+    <warnings>
+      <show t="boolean">true</show>
+      <timeout t="integer">10</timeout>
+      <log t="boolean">true</log>
+    </warnings>
+    <yesno_messages>
+      <show t="boolean">true</show>
+      <timeout t="integer">30</timeout>
+      <log t="boolean">true</log>
+    </yesno_messages>
+  </report>
+  <bootloader>
+    <loader_type>grub2</loader_type>
+    <global>
+      <activate>true</activate>
+      <boot_mbr>true</boot_mbr>
+      <gfxmode>auto</gfxmode>
+      <os_prober>true</os_prober>
+      <terminal>console serial</terminal>
+      <serial>serial --unit=1 --speed=115200 --word=8 --parity=no</serial>
+      <timeout t="integer">30</timeout>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>console=com2,115200 log_lvl=all guest_loglvl=all sync_console</xen_kernel_append>
+      <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
+    </global>
+  </bootloader>
+  <partitioning t="list">
+    <drive>
+      <device>/dev/sda</device>
```

Comment:
Better to avoid use device name directly ?

Reply:
I specify /dev/sda at here to install TW on the first disk and keep the 2nd disk in case of multiple systems on the machine.

Reply:
According to autoyast doc, better to use persistent device representation ?
```
device

    xxxxxxxxx

    You can use persistent device names via ID, like /dev/disk/by-id/ata-WDC_WD3200AAKS-75L9 or by-path, like /dev/disk/by-path/pci-0001:00:03.0-scsi-0:0:0:0.
```

Reply:
Can the device IDs or path for different machine be identicle? I am not quite familiar with it.

In this TW case, there are 2 machines in O3, maybe more in the future or be replaced by new machines. Only a static autoyast profile can be used in O3 infra, that is, no test settings can be referenced, so is it possible to use a device ID or path? @waynechen55  @alice-suse @tonyyuan1

Reply:
Don't use disk wwn in O3 as the infra is different from OSD as we discussed in meeting today.